packages/fx-core/src/plugins/resource/bot/plugin.ts

```diff
@@ -189,6 +189,22 @@ export class TeamsBotImpl {
     return ResultFactory.Success();
   }
 
+  public async updateArmTemplates(context: PluginContext): Promise<FxResult> {
+    this.ctx = context;
+    const bicepTemplateDir = path.join(getTemplatesFolder(), PathInfo.BicepTemplateRelativeDir);
+    const result: ArmTemplateResult = {
+      Configuration: {
+        Modules: {
+          botConfiguration: await fs.readFile(
+            path.join(bicepTemplateDir, PathInfo.ConfigurationModuleTemplateV2FileName),
+            ConstantString.UTF8Encoding
+          ),
+        },
+      },
+    };
```

Comment:
The return of updateArmTemplates is different from generateArmTemplates. If we add orchestration part in this return, what will happen? 
Should we replace ArmTemplateResult with a specified class?

Reply:
updateArmTemplates means will update moduleConfiguration.bicep files if need to regenerate.
if orchestration is needed in updateArmTemplates? if so, the orchestraction will do append files to provision.bicep or config.bicep files. this operation can support on solution as well